rebar.config

```diff
@@ -88,8 +88,8 @@
     deprecated_function_calls,
     deprecated_functions
 ]}.
-%% Set xref ignores for functions introduced in OTP 23
-{xref_ignores, [{code, get_doc, 1}, {shell_docs, render, 4}]}.
+%% Set xref ignores for functions introduced in OTP 23 & wrangler_handler
+{xref_ignores, [wls_handler, {code, get_doc, 1}, {shell_docs, render, 4}]}.
```

Comment:
You probably want something like:

%% Set xref ignores for functions introduced in OTP 23 & Wrangler modules:
{xref_ignores, [
    wls_handler,
    {code, get_doc, 1},
    {shell_docs, render, 4},
    {api_wrangler, start, 0},
    wrangler_handler,
    wls_code_lens,
    wls_highlight,
    wls_semantic_tokens,
    wls_code_actions,
    wls_execute_command_provider
]}.

Reply:
Thanks!

Reply:
@robertoaloi now?